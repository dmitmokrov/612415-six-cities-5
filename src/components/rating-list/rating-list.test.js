import React from 'react';
import renderer from 'react-test-renderer';
import RatingList from './rating-list';

const rating = `4`;

it(`Should RatingList render correctly`, () => {
  const tree = renderer
    .create(<RatingList
      rating={rating}
      onChange={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
