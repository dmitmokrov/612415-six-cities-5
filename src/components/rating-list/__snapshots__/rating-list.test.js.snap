// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RatingList render correctly 1`] = `
Array [
  <input
    checked={false}
    className="form__rating-input visually-hidden"
    id="5-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value="5"
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="5-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
  <input
    checked={false}
    className="form__rating-input visually-hidden"
    id="4-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value="4"
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="4-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
  <input
    checked={false}
    className="form__rating-input visually-hidden"
    id="3-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value="3"
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="3-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
  <input
    checked={false}
    className="form__rating-input visually-hidden"
    id="2-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value="2"
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="2-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
  <input
    checked={false}
    className="form__rating-input visually-hidden"
    id="1-stars"
    name="rating"
    onChange={[Function]}
    type="radio"
    value="1"
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="1-stars"
    title="perfect"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
]
`;
